apps/meteor/app/moderation/lib/permissions.ts

```diff
@@ -0,0 +1,23 @@
+import type { IPermission } from '@rocket.chat/core-typings';
+
+export const novice: IPermission['_id'][] = [
+	'view-joined-room',
+	'preview-c-room',
+	'leave-c',
+	'join-without-join-code',
+	'view-joined-room',
+	'view-history',
+];
+
+export const explorer: IPermission['_id'][] = [
```

Comment:
can you share the role => permissions on open for review? Purpose of a role and given permissions.

Reply:
Sure, I will do that. Thank you!